ios/REANodesManager.m

```diff
@@ -505,6 +520,25 @@ - (void)updateProps:(nonnull NSDictionary *)props
       ofViewWithTag:(nonnull NSNumber *)viewTag
            withName:(nonnull NSString *)viewName
 {
+  if (_viewRegistry[viewTag].superview == nil) {
```

Comment:
I think it is possible that we reach this stage and component is already mounted but still has entries in snapshot list (e.g. we run updates before and after mount + because we enqueue flushing with RCTExecuteOnMainQueue it may actually arrive later). As a consequence we may run older updates after these new ones. It'd be safer to check here if _componentUpdateBuffer contain updates